src/main/resources/application.properties

```diff
@@ -0,0 +1,40 @@
+############################################
+###       APPLICATION PARAMETERS         ###
+############################################
+
+spring.application.name=blacklist
+server.address=0.0.0.0
+server.port=8444
+
+# Database connection (mandatory)
+# Change the server timezone if neccessary
+spring.datasource.url=jdbc:mysql://127.0.0.1:3306/ah_blacklist?serverTimezone=UTC
+spring.datasource.username=blacklist
+spring.datasource.password=bl@ckl1st
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.ddl-auto=validate
+# use true only for debugging
+spring.jpa.show-sql=false
+spring.jpa.properties.hibernate.format_sql=true
+
+############################################
+###          CUSTOM PARAMETERS           ###
+############################################
+
+# Address to register into Service Registry with
+domain.name=127.0.0.1
+
```

Comment:
missing service.registry.address, service.registry.port. Only works for you because of the defaults